notebooks/byod.py

```diff
@@ -259,9 +272,69 @@ def create_cram_crai_table(table: str, listing: Iterable[str]):
     | NWD3      | NWD3.cram  | NWD3.crai | Adrian     | Zap       | Yes        |
 
     Note that the row for `NWD2` is missing from the combined table since it was not present in `diabetic_table`.
+    """
 
+with callysto.Cell("markdown"):
+    """
+    # Additional functions
+    To aid in the creation of your own data tables, we have provided some more functions for you to use and adapt.
     """
 
+with callysto.Cell("python"):
```

Comment:
Could we ask Brian for brief descriptions of each of these functions that can be included as comments?

Reply:
Yes! We can handle this in a separate PR.